packages/desktop-client/src/components/budget/MobileBudgetTableSmall.js

```diff
@@ -0,0 +1,1092 @@
+import React, { Component, memo, PureComponent, useState } from 'react';
```

Comment:
@matiss.ja I saw you comment and I totally agree that doing it that was is awful. The problem is that if I want to make it dynamic, you cannot just add a "if" statement in the mobiletable file. I need to check the viewport width whenever the size changes, meaning that I need to use the ResponsiveProvider code, and if I do that, I need to change everything else. Do you have any suggestions?

I can easily make this work for Iphone SE and below, but it will not be responsive.